x/crosschain/keeper/evm_hooks.go

```diff
@@ -327,8 +328,9 @@ func ValidateZrc20WithdrawEvent(event *zrc20.ZRC20Withdrawal, chainID int64) err
 	// The event was parsed; that means the user has deposited tokens to the contract.
 
 	if chains.IsBitcoinChain(chainID) {
-		if event.Value.Cmp(big.NewInt(0)) <= 0 {
-			return fmt.Errorf("ParseZRC20WithdrawalEvent: invalid amount %s", event.Value.String())
+		if event.Value.Cmp(big.NewInt(constant.BTCWithdrawalDustAmount)) < 0 {
+			return fmt.Errorf("ParseZRC20WithdrawalEvent: withdraw amount %s is less than minimum amount %d",
```

Comment:
NIT: We can define an error type for this  error

Reply:
https://github.com/zeta-chain/node/pull/2362/commits/28973eaa3086ba0cbfafcddf5ad78c09efa9386b